debug.log

```diff
@@ -0,0 +1 @@
+[1012/133010.920:ERROR:process_info.cc(375)] NtQuerySystemInformation SystemExtendedHandleInformation: The specified information record length does not match the length required for the specified information class. (0xc0000004)
```

Comment:
should this file be in .gitignore?

Reply:
I guess so, don't know how it got there.
Deleted & in .gitignore